### ISSUE TITLE
Search returns irrelevant content

### ISSUE THREAD

**Issue by nromyn** [MicrosoftDocs/feedback#743]
When searching for content, I frequently encounter a lot of irrelevant content, often but not always related to Dynamics documentation. 

**To Reproduce**
1. Go to docs.microsoft.com
2. Click on Search
3. Enter "monitor resource groups with azure monitor" (Without quotes)
4. View Results

**Expected behavior**
When I search for a specific title name, the article should be near the top of the results - in this case, the article is https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/resource-group-insights

**Screenshots**
Docs search results: https://pbs.twimg.com/media/DoqfWIJVAAAtLq9.jpg
Bing search results: https://pbs.twimg.com/media/DoqfW0cU4AAHttX.jpg

**Comment by EdisonDa** [comment]
@presleyy this is related to the site search, could you please take a look?

**Comment by EdisonDa** [comment]
@presleyy have you got some chance to review this?

**Comment by presleyy** [comment]
Yes, but I don't want to say that's bug just because of a single reason. This is something we are aware of related to the search relevancy from our search engine. A bunch of ideas have been initialized and under discussion. But for sure, we are going to improve the search relevancy.

**Comment by nromyn** [comment]
Any updates? Just did a search for "Azure Monitor" and got 285k results starting with Dynamics. It really renders the search useless without some kind of advanced querying capability

**Comment by presleyy** [comment]
We recently did some fixings to the site search crawler and indexer. We believe those changes could benefit this case. I tried searching "Azure Monitor" and got much better results than what shown in the screenshot. @nromyn would you please double check at your end?

**Comment by nromyn** [comment]
@presleyy It's still not great - if you test with the use case above there's still a fairly large distribution of Dynamics content and the article in question is still not located - even though it's the page title. 
![image](https://user-images.githubusercontent.com/13968908/56688702-5994b400-66a7-11e9-9587-bd262d2a9dad.png)
